source/configure/configuration-settings.rst

```diff
@@ -1924,9 +1924,9 @@ Enable Developer Mode
 
 **False**: Users are not alerted to Javascript errors.
 
-+-------------------------------------------------------------------------------------------------------------+
-| This feature's ``config.json`` setting is ``"EnableDeveloper": false`` with options ``true`` and ``false``. |
-+-------------------------------------------------------------------------------------------------------------+
++-----------------------------------------------------------------------------------------------------------------------------+
+| This feature's ``config.json`` setting is ``"ServiceSettings.EnableDeveloper": false`` with options ``true`` and ``false``. |
```

Comment:
Prefixing with `ServiceSettings.` isn't consistent with the rest of the documentation, and is technically incorrect: the setting isn't named `ServiceSettings.EnableDeveloper` but is actually nested as, `"ServiceSettings": { ..., "EnableDeveloper": false, ...}`, but we haven't traditionally expressed it this way in the documentation. (I definitely think we should find a way to fix this!)

Reply:
I appreciate this feedback, @lieut-data, thank you. We had started introducing this convention a few releases ago as new config settings have been introduced; however, I'm reverting those recent additions in this PR as a result of your input. 

I've added notes to an existing card on our docs backlog to revisit how best to communicate where, in a ``config.json`` file, users can find a given config setting. Any additional input or suggestions you have would be most welcome.